iCount/externals/star.py

```diff
@@ -200,6 +200,8 @@ def map_reads(reads, genome_index, out_dir, annotation='', multimax=10, mismatch
         '--outFilterMultimapNmax', '{:d}'.format(multimax),
         '--outFilterMismatchNmax', '{:d}'.format(mismatches),
         '--alignEndsType', 'EndToEnd',
+        ## Share memory
```

Comment:
Follow [PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments) for inline comments.